### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by neeru24**
Done with the changes!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-#### A <b> responsive and dynamic website </b> to showcase the best gaming accessories for every gamer!
+A <b> responsive and dynamic website </b> to showcase the best gaming accessories for every gamer!
 
 
 # ✨ Features
-#### 🗂️ Dynamic Navigation Menu: Toggle visibility, designed for seamless use across all screen sizes.
-#### 🛒 Product Display: View gaming products with prices, discounts, and a "Buy Now" option.
-#### 🌐 Social Media Links: Instant access to your favorite platforms like Facebook, Twitter, and Instagram.
-#### 📱 Responsive Design: Enjoy smooth navigation whether you're on mobile, tablet, or desktop!
+🗂️ <b>Dynamic Navigation Menu</b>: Toggle visibility, designed for seamless use across all screen sizes. <br> <br>
```
Comment (swaraj-das) [inline README.md:6]: don't use <br> tag for next line...use double space.

**Comment by neeru24** [comment]
> Remove the br tags then I will merge it.

Wait. Let me do so

**Comment by neeru24** [comment]
> Remove the br tags then I will merge it.

Kindly Review again!

**Review by swaraj-das — APPROVED** [review]
That's good.

**Comment by neeru24** [comment]
> That's good.

Thank you so much for guiding me. It's my first time contributing to open source here. Really grateful for your support 👍🙏

**Pull request merged**